hknweb/events/views.py

```diff
@@ -42,10 +42,12 @@ def show_details(request, id):
     
     event = get_object_or_404(Event, pk=id)
 
-    rsvp = Rsvp.objects.filter(user=request.user, event=event).exists()
+    rsvpd = Rsvp.objects.filter(user=request.user, event=event).exists()
+    other_rsvps = Rsvp.objects.filter(event=event).exclude(user=request.user)
     context = {
         'event': event,
-        'rsvp': rsvp,
+        'rsvpd': rsvpd,
+        'other_rsvps': other_rsvps,
```

Comment:
Beyond the nifty (oh look I rsvp'd) reason of listing "You" as an rsvp, is there any other reason to not explicitly list the person's name / email (and include the request.user in the query)?
I imagine if csec wants a record of everyone at an event (or any other officer), they need to copy + paste the list and then add themselves, esp. for reimbursements.

Reply:
Csec could user the admin page

Reply:
csec, perhaps, but other officers? So far we haven't given most officers access to the admin page, nor have we decided to do so.
Admittedly a small nitpick, that doesn't really affect the decision to merge.

Reply:
Agreed, just display the name